ldap/servers/slapd/back-ldbm/db-mdb/mdb_verify.c

```diff
@@ -17,5 +17,6 @@
 int
 dbmdb_verify(Slapi_PBlock *pb)
 {
+    slapi_log_err(SLAPI_LOG_WARNING, "dbmdb_verify", "With lmdb, db_verify feature is meaningless and always successful.\n");
```

Comment:
Very minor issue with grammar:

    With lmdb, db_verify feature is meaningless and is always successful